README.tfdoc.hcl

```diff
@@ -226,42 +251,69 @@ section {
           }
         }
       }
-    }
-  }
 
-  section {
-    title   = "Module Outputs"
-    content = <<-END
+      section {
+        title = "Module Configuration"
+
+        variable "module_enabled" {
+          type        = bool
+          default     = true
+          description = <<-END
+            Specifies whether resources in the module will be created.
+          END
+        }
+
+        variable "module_depends_on" {
+          type           = list(dependency)
+          description    = <<-END
+            A list of dependencies. Any object can be _assigned_ to this list to define a hidden external dependency.
+          END
+          readme_example = <<-END
+            module_depends_on = [
+              google_network.network
+            ]
+          END
+        }
+      }
+      section {
+        title   = "Module Outputs"
+        content = <<-END
       The following attributes are exported in the outputs of the module:
     END
 
-    output "iam" {
-      type        = bool
-      description = <<-END
-        All attributes of the created `iam_binding` or `iam_member` or
-        `iam_policy` resource according to the mode.
+        output "iam" {
+          type        = object(iam)
+          description = <<-END
+        All attributes of the created 'iam_binding' or 'iam_member' or 'iam_policy' resource according to the mode."
       END
-    }
-  }
+        }
+      }
 
-  section {
-    title = "External Documentation"
+      section {
```

Comment:
indentation looks wrong now?

Reply:
terradoc would cry if it was? it was crying before and fixed it